dtcInterfaceLib/CFOandDTC_Registers.cpp

```diff
@@ -141,6 +141,18 @@ std::string DTCLib::CFOandDTC_Registers::ReadDesignDate(std::optional<uint32_t>
 	return o.str();
 }
 
+/// <summary>
+/// Read the DTC/CFO firmware flavour
+/// </summary>
+/// <returns>Returns the firmware flavour: C for CFO, D for DTC, E for CRVDTC</returns>
+std::string DTCLib::CFOandDTC_Registers::ReadDesigFlavour(std::optional<uint32_t> val)
```

Comment:
I think we should trade "ReadDesignFlavour" for something like "isCRVDTCDesignFlavour()," "isNonCRVDTCDesignFlavour()," and "isCFODesignFlavour()" which each just returns true/false. This way the implementation is contained within getCFOandDTCRegisters and the interpretation is simple by only using booleans.